docs/guides/middleware.md

```diff
@@ -5,28 +5,43 @@ Middleware is a Video.js feature that allows interaction with and modification o
 ## Table of Contents
 
 * [Understanding Middleware](#understanding-middleware)
+  * [setSource](#setsource)
+  * [setTech](#setTech)
   * [Middleware Setters](#middleware-setters)
   * [Middleware Getters](#middleware-getters)
   * [Middleware Mediators](#middleware-mediators)
   * [Termination and Mediators](#termination-and-mediators)
 * [Using Middleware](#using-middleware)
   * [Terminating Mediator Methods](#terminating-mediator-methods)
-* [setSource](#setsource)
 
 ## Understanding Middleware
 
-Middleware are functions that return an object with methods matching those on the `Tech`. There are currently a limited set of allowed methods that will be understood by middleware. These are: `buffered`, `currentTime`, `setCurrentTime`, `duration`, `seekable`, `played`, `play`, `pause` and `paused`.
+Middleware are functions that return an object, a class instance, a prototype, e.c.t, scoped to the Player with methods matching those on the `Tech`. There are currently a limited set of allowed methods that will be understood by middleware. These are: `buffered`, `currentTime`, `setCurrentTime`, `duration`, `seekable`, `played`, `play`, `pause` and `paused`. These allowed methods are split into three categories: [getters](#middleware-getters), [setters](#middleware-setters), and [mediators](#middleware-mediators).
```

Comment:
Is e.c.t a correct stylization? I usually see etc.

Reply:
whoops, good catch. I often make this mistake